src/app/(homepage)/Demo.js

```diff
@@ -2,30 +2,38 @@ import OosStepper from "@/components/OosStepper";
 import { Box, Container } from "@mui/material";
 import Typography from "@mui/material/Typography";
 
-function DemoHeader({sx}) {
+
+function DemoHeader({ sx, subheading1, heading1, heading2}) {
   return (
-    <Box sx={{textAlign: "center", maxWidth: 735, ...sx}}>
-      <Typography variant="subheading">
-        Directly from the calendar
-      </Typography>
-      <Typography mt={2} variant="h2">
-        Discover the ease of booking activities without the text and email tennis
-      </Typography>
+    <Box sx={{ textAlign: "center", maxWidth: 735, ...sx }}>
+      {subheading1 && <Typography variant="subheading">{subheading1}</Typography>}
+      {heading1 && <Typography mt={2} variant="h2">{heading1}</Typography>}
     </Box>
-  )
+  );
 }
 
-export default function Demo() {
-  const steps = [
-    "Find the activities your child will love",
-    "Add them to child’s calendar",
-    "Review and then pay for all in one click",
-  ];
+function Demo(props) {
```

Comment:
General props passing is a bit less descriptive, as it does not allow to see what args are passed.
So I'd use this form: { sx, subheading1, heading1, heading2}